borg/lrucache.py

```diff
@@ -1,42 +1,41 @@
-class LRUCache(dict):
```

Comment:
hmm, you removed subclassing dict. don't you lose all the dict methods that way?